.github/workflows/xcode.yml

```diff
@@ -0,0 +1,72 @@
+name: Xcode
```

Comment:
would make sense to keep inside `macos.yml`
also perhaps merging it with the normal workflow, so we don't build twice

Reply:
Moved inside `macos.yml`.
However, it runs under macOS-14 (arm64), not macOS-13 (intel), so it is not merged into the normal workflow.